README.md

```diff
@@ -29,6 +29,19 @@ This is a [Node.js](https://nodejs.org/en/) module available through the
 $ npm install mime-types
 ```
 
+## Note on MIME Type Data and Semver
+
+This package considers the programmatic api as the semver compatibility. Additionally, the package which provides the MIME data
+for this package (`mime-db`) *also* considers it's programmatic api as the semver contract. This means the MIME type resolution is *not* considered
+in the semver bumps.
+
+In the past the version of `mime-db` was pinned to give two decision points when adopting MIME data changes. This is no longer true. We still update the
+`mime-db` package here as a `minor` release when necessary, but will use a `^` range going forward. This means that if you want to pin your `mime-db` data
+you will need to do it in your application. While this expectation was not set in docs until now, it is how the pacakge operated, so we do not feel this is
+a breaking change.
```

Comment:
This sounds good to me. I think the argument is that if you didn't pin already, you expected updates on resolution automatically, and if you did pin, you didn't. This breaks the "you didn't" case, but since those users are pinned I agree that as long as the release notes are _very_ clear I think this is reasonable.

I still think it's _clearer_ to consider a major version that changes the way the API works and this change at the same time, but it's certainly _easier_ (on both us and users) to consider this non-breaking.